Embedded/setup.in.py

```diff
@@ -24,8 +34,8 @@
         "@CMAKE_SOURCE_DIR@/ThirdParty/rapidjson",
         "@CMAKE_SOURCE_DIR@/Distributed/os",
     ],
-    library_dirs=pa.get_library_dirs() + ["@CMAKE_CURRENT_BINARY_DIR@", "."],
-    runtime_library_dirs=pa.get_library_dirs() + ["$ORIGIN/../../"],
+    library_dirs=pa.get_library_dirs() + ["@CMAKE_CURRENT_BINARY_DIR@", ".", omniscidbe_root],
+    runtime_library_dirs=pa.get_library_dirs() + ["$ORIGIN/../../", omniscidbe_root],
```

Comment:
why is this needed again?

Reply:
This is required for building Python module omniscidbe from conda recipe. Notice that `libDBEngine.so` is provided by another conda package (`omniscidbe`) and is not available as `Embedded/`libDBEngine.so` that setup.py otherwise assumes.

There are actually two possible approaches to specify the path to `libDBEngine.so`:
1. `$CONDA_PREFIX/lib`
2. `$OMNISCI_ROOT_PATH/lib`

In this PR, option 2. is used as a more general one - it would be functional when using some non-conda environment.

Reply:
Could you please clarify the file location structure for the CF package build?

Reply:
Sure. All paths will be shown relative to conda environment `$CONDA_PREFIX`:

1. `omniscidb-common` defines the following paths
    ```
    opt/omnisci/bin/{calcite, omnisci}-*.jar
    opt/omnisci/QueryEngine/RuntimeFunctions.bc
    ...
    ```
2. `omniscidb` defines
    ```
    opt/omnisci/bin/{omnisci_server,omnisci_initdb,omnisql,...}
    ```
3. `omniscidbe` defines
    ```
    opt/omnisci/lib/libDBEngine.so
    lib/libDBEngine.so -> ../opt/omnisci/lib/libDBEngine.so
    ```
4. `pyomniscidbe` for Python 3.8 defines
    ```
    lib/python3.8/site-packages/omniscidbe.cpython-38-x86_64-linux-gnu.so
    ```

Reply:
Thank you. Looks good